app/controllers/admin/protocol_drugs_controller.rb

```diff
@@ -56,7 +56,9 @@ def protocol_drug_params
     params.require(:protocol_drug).permit(
       :name,
       :dosage,
-      :rxnorm_code
+      :rxnorm_code,
+      :drug_category,
+      :stock_tracked
```

Comment:
From the PRD, it seems like we want this to restrict what is seen on the dashboard. Does it also restrict the drugs that can be tracked? 

If we don't want to restrict the data-entry, maybe we should rename the field to reflect viewing, or reporting, something on those lines. `stock_tracked_on_dashboard`?

Reply:
Vikram explained over a conversation. Seems like we want to keep the two concerns of viewing-on-dashboard, and collecting-data as the same thing for now.

We can revisit splitting these two concerns later, but that would involve a data migration. 

cc @harimohanraj89 @kitallis

Reply:
Sounds good 👍